platform/darwin/src/NSExpression+MGLAdditions.mm

```diff
@@ -553,6 +553,10 @@ + (NSExpression *)lineProgressVariableExpression {
     return [NSExpression expressionForVariable:@"lineProgress"];
 }
 
++ (NSExpression *)featureAccumulatedVariableExpression {
```

Comment:
Please add this method to the header https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/src/NSExpression+MGLAdditions.h#L93